### PR TITLE
Add transceiver test for set session description

### PR DESCRIPTION
**Pull request by soareschen**
This is added to a new file RTCPeerConnection-setDescription-transceiver.html in attempt to better organize the tests for setting session description.

### CODE REVIEW

File: webrtc/RTCPeerConnection-setDescription-transceiver.html [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    const pc = new RTCPeerConnection();
+    const transceiver = pc.addTransceiver('audio');
+    assert_equals(transceiver.mid, null);
+
+    return pc.createOffer()
+    .then(() => {
+      assert_equals(transceiver.mid, null,
+        'Expect transceiver.mid to still be null after createOffer');
+
+      return pc.setLocalDescription(offer);
```
Comment (rwaldron) [inline webrtc/RTCPeerConnection-setDescription-transceiver.html]: `offer` is not defined anywhere in this scope

File: webrtc/RTCPeerConnection-setDescription-transceiver.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+        const [ transceiver2 ] = transceivers;
+
+        assert_equals(typeof transceiver2.mid, 'string',
+          'Expect transceiver2.mid to be set');
+
+        assert_equals(transceiver1.mid, transceiver2.mid,
+          'Expect transceivers of both side to have the same mid');
+
+        assert_equals(offer.sdp.includes(`\r\na=mid:${transceiver2.mid}`), true,
```
Comment (rwaldron) [inline webrtc/RTCPeerConnection-setDescription-transceiver.html]: `offer` is not defined anywhere in this scope

**Review by rwaldron — CHANGES_REQUESTED** [review]
Issues with test code that need to be addressed

File: webrtc/RTCPeerConnection-setDescription-transceiver.html (hunk 2 above)
Comment (soareschen) [inline webrtc/RTCPeerConnection-setDescription-transceiver.html]: Thanks for spotting. I have rebased and fixed the errors.

**Review by rwaldron — APPROVED** [review]

**Pull request merged**